zerver/views/realm.py

```diff
@@ -121,3 +122,10 @@ def update_realm(
             data['signup_notifications_stream_id'] = signup_notifications_stream_id
 
     return json_success(data)
+
+def check_subdomain_available(request: HttpRequest, subdomain: Text) -> HttpResponse:
```

Comment:
This function probably should have `@require_GET` or `require_POST` on it; whichever is the way we query this endpoint.